tests/markers.py

```diff
@@ -19,5 +19,5 @@
 import pytest
 
 
-circleci_only = pytest.mark.skipif(condition=('CIRCLECI' not in os.environ),
-                                   reason='Only run on CircleCI')
+circleci_only = pytest.mark.skipif(condition=('CIRCLECI' not in os.environ), reason='Only run on CircleCI')
+skip_circleci = pytest.mark.skipif(condition=('CIRCLECI' in os.environ), reason='Only run on CircleCI')
```

Comment:
`skip_on_circleci` would sound a little better to me, but I don't insist on it.

Reply:
:heavy_check_mark: - Yes, this is more clear.